gulpfile.babel.js

```diff
@@ -154,6 +154,18 @@ gulp.task('html', () => {
       removeStyleLinkTypeAttributes: true,
       removeOptionalTags: true
     })))
+    .pipe($.if('*.html', $.minifyInline({
+            js: {
+                output: {
+                    comments: false
+                }
+            },
+            jsSelector: 'script[type!="text/x-handlebars-template"]',
```

Comment:
This either needs to be *removed* or we need to do something custom like the cssSelector one. We should not have ignores for things our repo doesn't use.